R/tm_g_scatterplotmatrix.R

```diff
@@ -147,38 +147,44 @@ ui_g_scatterplotmatrix <- function(id, ...) {
         )
       )
     ),
-    forms = teal::get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = args$pre_output,
     post_output = args$post_output
   )
 }
 
-srv_g_scatterplotmatrix <- function(id, datasets, reporter, variables, plot_height, plot_width) {
+srv_g_scatterplotmatrix <- function(id, data, reporter, filter_panel_api, variables, plot_height, plot_width) {
```

Comment:
Deslecting the dataset in the example app renders a red error:
![Screenshot from 2022-08-29 18-06-32](https://user-images.githubusercontent.com/86970066/187245340-9fbc3a8d-431c-4aab-9127-9b156a7bbe86.png)